main.go

```diff
@@ -49,7 +49,10 @@ func init() {
 
 func main() {
 	var debugLogging bool
+	var runtime string
+
 	flag.BoolVar(&debugLogging, "debugLogging", false, "Log debug messages")
+	flag.StringVar(&runtime, "runtime", "docker", "Container runtime to be used")
```

Comment:
I am fine with this being a string rather than bool, but we should validate it is an expected value (`containerd` or `docker`).

Reply:
I think either we skip --runtime so that default value will kick in or we should use --runtime=<value>.  while testing our initial draft was PR was just having --runtime. WMCO container didn't come up due to the parameter value being left out. We can test it one more time by removing it and make sure the default value kicks in